build/Dockerfile.helper

```diff
@@ -1,29 +1,23 @@
-# docker buildx build --platform linux/amd64,linux/arm64 -t quay.io/codefresh/golang-ci-helper:1.21.12 -f Dockerfile.helper .
```

Comment:
this is just the dockerfile for our golang-ci-helper image. not being built automatically anywhere.